percpu_macros/src/arch.rs

```diff
@@ -103,6 +111,26 @@ pub fn gen_read_current_raw(symbol: &Ident, ty: &Type) -> proc_macro2::TokenStre
         )
     };
 
+    // https://loongson.github.io/LoongArch-Documentation/LoongArch-Vol1-EN.html#common-memory-access-instructions
+    let la64_op = match ty_str.as_str() {
+        "bool" => "ld.bu",
+        "u8" => "ld.bu",
+        "u16" => "ld.hu",
+        "u32" => "ld.wu",
+        "u64" => "ld.d",
+        "usize" => "ld.d",
+        _ => unreachable!(),
+    };
+    let la64_asm = quote! {
+        ::core::arch::asm!(
+            "la.abs {0}, {VAR}",
+            "add.d {0}, {0}, $r21",
```

Comment:
The following two instructions can be simplified into one instruction using `ldx.{b/h/w/d}`. e.g., `add.d $r0, $r0, $r21; ld.d $r0, $r0, 0` => `ldx.d $r0, $r0, $r21`.